pkgs/by-name/po/poetry/unwrapped.nix

```
@@ -45,14 +47,14 @@ buildPythonPackage rec {
     owner = "python-poetry";
     repo = "poetry";
     tag = version;
-    hash = "sha256-RpAoADxZmH9hQSEjufLBoKJsxIc74RnRxZB3RVNk/iE=";
+    hash = "sha256-2u0idmnXY0FNvc8peDMWIFAeGH/xdIfMrMErUdxc0UA=";
   };
 
   patches = [
     # https://github.com/python-poetry/poetry/pull/9939
     (fetchpatch2 {
-      url = "https://github.com/python-poetry/poetry/commit/89c0d02761229a8aa7ac5afcbc8935387bde4c5b.patch?full_index=1";
-      hash = "sha256-YuAevkmCSTGuFPfuKrJfcLUye1YGpnHSb9TFSW7F1SU=";
+      url = "https://github.com/python-poetry/poetry/pull/9939/commits/c2387ff3c878ab608d7616e4984fc01c4226416c.patch?full_index=1";
```

```suggestion
      url = "https://github.com/python-poetry/poetry/commit/c2387ff3c878ab608d7616e4984fc01c4226416c.patch?full_index=1";
```
Otherwise the URL will result in a 404 if I force push the PR's branch.